addons/website_sale/views/templates.xml

```diff
@@ -712,6 +712,28 @@
     </t>
     </template>
 
+    <template id="alternative_products" name="Alternative Products" inherit_id="website_sale.product" active="True">
+        <div itemprop="description" position="after">
+            <div class="oe_structure oe_structure_solo oe_unremovable oe_unmovable" id="oe_structure_website_sale_recommended_products" t-ignore="true" t-if="product.alternative_product_ids">
```

Comment:
Actually yes we need the "branding" on that div otherwise attributes on the dynamic snippet's section are never saved.
At first I had a diff in the web_editor code to allow that but I felt like it was a bit too dangerous. (due to [this](https://github.com/odoo/odoo/blob/master/addons/web_editor/models/ir_ui_view.py#L130))

I'm all ears for other solutions though.